src/org/daisy/dotify/formatter/impl/volume/TocSequenceEventImpl.java

```diff
@@ -145,6 +145,12 @@ public BlockSequence getBlockSequence(FormatterContext context, DefaultContext v
 						fsm.appendGroup(volumeEnd);
 					}
 				}
+				{
```

Comment:
Do we need the code block? Or should this be an if statement?

Reply:
The block is for variable scoping.